src/Microsoft.Extensions.PlatformAbstractions.Internal/Native/NativeMethods.Unix.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Extensions.PlatformAbstractions.Internal.Native
+{
+    internal static partial class NativeMethods
+    {
+        public static class Unix
+        {
+            public unsafe static string GetUname()
```

Comment:
There's an issue with the InteropServices package in net4x where the package fails to work at runtime unless the application explicitly includes the runtime package. https://github.com/dotnet/corefx/issues/7863 fixes it for RTM.